akka/src/main/scala/cakesolutions/kafka/akka/KafkaProducerActor.scala

```diff
@@ -10,42 +10,112 @@ import scala.concurrent.Future
 import scala.reflect.runtime.universe.TypeTag
 import scala.util.{Failure, Success, Try}
 
+/**
+  * An actor that wraps [[KafkaProducer]].
+  *
+  * The actor takes incoming (batches of) Kafka records, writes them to Kafka,
+  * and responds to sender once the messages have been written.
+  *
+  * [[KafkaProducerActor]] is not tied to any specific topic,
+  * but it's message serializers have to be specified before it's used.
```

Comment:
This is really pedantic but I don't think there should be an apostrophe in the first "it's".